lib/rspec/matchers/built_in/be.rb

```diff
@@ -75,7 +75,7 @@ module BeHelpers
       private
 
         def args_to_s
-          @args.empty? ? "" : parenthesize(inspected_args.join(', '))
+          @args.empty? ? '' : parenthesize(inspected_args.join(', '))
```

Comment:
If you run `bundle exec rubocop lib` you can see what our rubocop rules enforce.